configuration/customizations/Sub-brands

```diff
@@ -0,0 +1,55 @@
+---
+title: "Sub-brands"
+description: "Sub-brands allow you to manage users across all your varying apps, websites and products"
+---
+
+Introducing sub-brands, one of Rownd’s latest and greatest features focused on allowing you to expand your portfolio without partitioning where you manage your users. Sub-brands are extremely unique, easy to use, and can save you and your development team so much time.
+
+
+#### What are sub brands?
+
+A sub-brand is a variation of your Rownd application. Sub-brands can have their own key, theme, customizations, and even associated automations. Sub-brands allow you to see which apps and brands your users are interacting with, without having to switch between apps. Users maintain their account and profile information across the sub-brands they are signing into.
+
+
+#### Why might you enable sub-brands?
+
+Sub-brands are most often used when customers have a multitude of brands or apps that they want users to stay signed in across. Here are a few examples and use cases:
+
+- League or collection of apps: For example, a sports league wants sign-ins for their rewards program, but each team has its own application and requirements. In this scenario, the larger Rownd app would represent the league, and each team would be its own sub-brand. This way, each team could still have its logo and branding colors, as well as its sign-in method configuration, while at the larger app level, the league can see which users are interacting with which teams (sub-brands), and users can easily sign in across team apps.
+
+- Parent brand scenario: When one company owns multiple brands (Example: Walt Disney Company owning ESPN, Hulu, Disney +, ABC, etc.), they’d want to know which customers are interacting with which apps, without those users having a separate account for each app. By using sub-brands, the parent brand can better understand their users in seeing where they are visiting.
+
+- Testing scenario: You can also use sub-brands as a way of testing your Rownd flow without creating a new app.
```

Comment:
```suggestion
- Testing scenario: You can also use sub-brands as a way of testing your Rownd flow without impacting production users. Since you can attach a specific app key to a sub-brand, you can create a sub-brand with changes that you'd like to change and use its app key in a separate testing environment to validate the changes.
```